src/content/learn/react-developer-tools.md

```diff
@@ -64,13 +63,13 @@ yarn global add react-devtools
 npm install -g react-devtools
 ```
 
-Next open the developer tools from the terminal.
+Poté nástroje otevřte v Terminálu:
 ```bash
 react-devtools
 ```
 
-It should connect to any local React Native app that's running.
+Nástroje se nyní sami spojí s vaší React Native aplikací, která běží v lokálním prostředí.
 
-> Try reloading the app if developer tools doesn't connect after a few seconds.
+> Zkuste obnovit aplikaci pokud se nástroje po několika sekundách nepřipojí sami.
```

Comment:
I think, there should be y. I found this description https://www.pravopisne.cz/2015/02/samy-x-sami/.
```suggestion
> Zkuste obnovit aplikaci pokud se nástroje po několika sekundách nepřipojí samy.
```